### PR TITLE
Modal replacement (#335)

### PR DESCRIPTION
**Pull request by RajChaudhry**
### Before submitting this PR, please use netlify to make sure:
- [ ] The app looks okay on web
- [ ] The app looks okay on phone
- [ ] Filter by category still works

### If you might be concerned about these things, also check:
- [ ] The app looks okay in ipad
- [ ] Search still works

just in case we decide to add the alert box, I added in the functionality, but it currently does not show up. To make it do so, just change line 151 from toggle={this.removalConfirmed} to toggle={this.removeItem} . addresses #335. 

It is difficult to remove resources quickly in our current build because the SW has to confirm they want to remove each resource individually. This can be solved by removing the resource with one click, and informing via an alert box on the top of the screen that it was removed rather than asking for permission every time.

### CODE REVIEW

File: src/components/AdminPage/CardGrid.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export class CardGrid extends Component {
 
   render() {
     const sortOptions = [
-      {key: 'Alphabetically', sort: this.sortByAlphabet, disabled: false}
+      {key: 'A-Z', sort: this.sortByAlphabet, disabled: false}
```
Comment (timtheguy) [inline src/components/AdminPage/CardGrid.js]: Why are you making this change here?

File: src/components/AdminPage/AdminPageLayout.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ export const SearchAndSortWrapper = styled("div")`
     display: grid;
     grid-template-columns: auto auto;
     justify-content: space-between;
+    border: 3px solid #73AD21;
```
Comment (timtheguy) [inline src/components/AdminPage/AdminPageLayout.js]: What is this change related to?

File: src/components/AdminPage/CardGrid.js (hunk 1 above)
Comment (RajChaudhry) [reply src/components/AdminPage/CardGrid.js]: Shoot, got my git branches tangled up a bit. This was supposed to be it's own PR, addressing #320 , in order to make that label consistent between pages. I will take this out and make it it's own PR

File: src/components/AdminPage/AdminPageLayout.js (hunk 2 above)
Comment (RajChaudhry) [reply src/components/AdminPage/AdminPageLayout.js]: Debugging some CSS stuff, again, got my git branches tangled up. Will fix and submit again.

Comment (timtheguy) [reply src/components/AdminPage/AdminPageLayout.js]: Got it, looks good!
